routers/web/repo/wiki.go

```diff
@@ -178,23 +179,17 @@ func wikiContentsByName(ctx *context.Context, commit *git.Commit, wikiName wiki_
 }
 
 func renderViewPage(ctx *context.Context) (*git.Repository, *git.TreeEntry) {
-	wikiRepo, commit, err := findWikiRepoCommit(ctx)
+	wikiGitRepo, commit, err := findWikiRepoCommit(ctx)
 	if err != nil {
-		if wikiRepo != nil {
-			wikiRepo.Close()
-		}
 		if !git.IsErrNotExist(err) {
 			ctx.ServerError("GetBranchCommit", err)
 		}
 		return nil, nil
 	}
 
```

Comment:
Isn't there a `defer wikiGitRepo.Close()` missing?

Reply:
That's the power our new framework.

We can easily use gitrepo now without keeping copying-pasting `defer gitRepo.Close()`

![image](https://github.com/user-attachments/assets/f5ff974f-cb55-44f1-9c24-3f9e68cc4bbe)